### ISSUE TITLE
Can uftrace track the elf process before loading the main function?

### ISSUE THREAD

**Issue by lh233** [namhyung/uftrace#1850]
How to use uftrace to locate the problem in my program, as it takes a long time to start before the main function

**Comment by namhyung** [comment]
Do you mean initializer functions in your binary or start up functions in the C library?

**Comment by namhyung** [comment]
No, it cannot trace the C library because it has all the tricky parts. :(

**Comment by lh233** [comment]
OK，I get it. Thanks.
